build/get_concurrent_jobs.py

```diff
@@ -0,0 +1,102 @@
+#!/usr/bin/env python3
+# Copyright 2019 The Fuchsia Authors. All rights reserved.
```

Comment:
Is this copyright script meant to remain as-is? Presumably, this script is from the Fuchsia buildroot then?

Reply:
I see, it's [this](https://fuchsia.googlesource.com/fuchsia/+/HEAD/build/toolchain/get_concurrent_jobs.py) but with added Windows support. I'll review the delta.

Reply:
Yeah, it's from the Fuchsia buildroot. It looks like for other files from the Fuchsia buildroot we're leaving the Fuchsia copyright header.